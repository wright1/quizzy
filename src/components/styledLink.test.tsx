import { render } from '@testing-library/react'
import { BrowserRouter as Router } from 'react-router-dom'
import StyledLink from './styledLink';

describe('Button', () => {
    it('should match snapshot', () => {
        const { container } = render(
        <Router>
        <StyledLink to="/">Take a Break</StyledLink>
        </Router>
        )
        expect(container).toMatchSnapshot()
    })

})