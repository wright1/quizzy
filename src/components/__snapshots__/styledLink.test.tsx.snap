// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should match snapshot 1`] = `
<div>
  <a
    class="sc-bdvvtL fTQGOF"
    href="/"
  >
    Take a Break
  </a>
</div>
`;
